src/index.html

```diff
@@ -6,13 +6,47 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Please ensure that the SCSS is compiled to a CSS file and update the `href` to point to the resulting CSS file.